examples/templates/api/change-algorithm-suite.md

```diff
@@ -0,0 +1,133 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Changing the algorithm suite example
+
+Implementations of this example MUST follow the rules defined in
+[Example Templates](../../../examples.md#example-templates).
```

Comment:
This applies here and in other documents in this PR.

I think this path isn't resolving properly.

Reply:
I think these might just need to be 

[Example Templates](../../examples.md#example-templates).

Reply:
doh; that is correct; I copy-pasted that from the raw keyring examples and they have at least one extra directory layer x_x